internal/interface/handlers/attachment_handler.go

```diff
@@ -80,5 +80,13 @@ func (h *AttachmentHandler) CreateAttachments(c *fiber.Ctx) error {
 // @Failure 500 {object} response.Response
 // @Router /attachments/{attachment_id} [delete]
 func (h *AttachmentHandler) DeleteAttachment(c *fiber.Ctx) error {
-	return nil
+	attachmentID := strings.Trim(c.Params("attachment_id"), " ")
+
+	if err := h.attachmentUsecase.DeleteAttachment(attachmentID); err != nil {
+		resp := response.NewResponseFactory(response.ERROR, err.Error())
+		return resp.SendResponse(c, fiber.StatusInternalServerError)
```

Comment:
you can use this:
```go
resp.SendResponse(c, err.HttpCode)
```
err is of type AppError (we custom wrote this so that it can pass status codes from usecase to handlers)